rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc

```diff
@@ -0,0 +1,68 @@
+= Activity Library
+
+The Activity Library enables you to reuse certain activities that repeat during the automation of business processes, a login for example, and avoid creating the same workflow multiple times.
+
+Use the Activity Library Explorer to edit, copy, or delete activities and search for activities in your organization's library based on Activity Tags, which you can create and assign to the activities to further categorization and filtering capabilities. 
+
+== Before you Begin 
+
+//Update this when IAM is implemented.
+* To open and to use the Activity Library, you need the *Activity Library Open* and *Activity Library Administration* privileges. 
++
+For more information about assigning user privileges, see xref:rpa-manager::usermanagement-manage.adoc#assign-privileges-to-a-user[Assign Privileges to a User].
+
+== Upload an Activity to the Activity Library
+
+To upload an activity to the Activity Library:
+
+. In RPA Builder, open a process for editing. 
+. Click an activity to select it and open the context pad. 
+. In the context pad, click *Add to Activity Library* (image:add-to-activity-library-icon.png["The Add to Activity Library Icon.", 2%, 2%])
```

Comment:
For all of these alt texts that aren't complete sentences - remove the terminating punctuation.

Reply:
And probably the "The"s